### PR TITLE
bob v2.6.0 (new formula)

### PR DESCRIPTION
**Pull request by Amar1729**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

New formula for the [`bob`](https://github.com/MordechaiHadad/bob), a version manager for neovim.

https://github.com/MordechaiHadad/bob/issues/143

### CODE REVIEW

**Review by ZhongRuoyu — APPROVED** [review]

File: Formula/b/bob-nvim.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class BobNvim < Formula
```
Comment (chenrui333) [inline Formula/b/bob-nvim.rb:1]: why not just `bob`?

Comment (chenrui333) [reply Formula/b/bob-nvim.rb:1]: note, other package managers are using `bob`, https://repology.org/project/bob/versions

Comment (p-linnane) [reply Formula/b/bob-nvim.rb:1]: I agree, this should be `bob`.

Comment (Amar1729) [reply Formula/b/bob-nvim.rb]: Cool, renamed. Wasn't sure if `bob` alone was unique enough.

**Review by p-linnane — APPROVED** [review]
Thanks @Amar1729!

File: Formula/b/bob-nvim.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/b/bob-nvim.rb]: I think `bob` is pretty unique (so is [alice](https://repology.org/project/alice/versions))

**Pull request merged**